game/tests.py

```
@@ -1052,3 +1052,141 @@ def test_protected_endpoint(self):
+    """Test that abandoned OTP registrations do not lock out usernames."""
+
+    def test_stale_ghost_is_replaced_on_re_register(self):
+        """A ghost account older than 1 hour should be auto-deleted
+        when a new user registers with the same username."""
+        from django.utils import timezone
+        from datetime import timedelta
+
+        ghost = User.objects.create_user(
+            username='ghostuser',
+            email='ghost@example.com',
+            password='OldPass123!',
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "password"

(S106)